src/services/resources-getter.js

```diff
@@ -57,7 +58,12 @@ function ResourcesGetter(model, options, params) {
 
     _.each(params.filter, (values, key) => {
       if (key.indexOf(':') !== -1) {
-        key = `$${key.replace(':', '.')}$`;
+        const [associationName, fieldName] = key.split(':');
+        const schemaField = schema.fields.find(field => field.field === associationName);
+        const [tableName] = schemaField.reference.split('.');
+        const associationSchema = Schemas.schemas[tableName];
+        const belongsToColumnName = Orm.getColumnName(associationSchema, fieldName);
+        key = `$${associationName}.${belongsToColumnName}$`;
```

Comment:
Ok for the fix 💪 but this code logic is already present in `query-builder.js` so your should find a way to factorize it 👍